AppOne/AppOne/View/HomeVC.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  HomeVC.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 19.11.22.
+//
+
+import UIKit
+
+final class HomeVC: UIViewController {
+    private lazy var brandsCollection: UICollectionView = {
+        let collectionV = UICollectionView(
+            frame: view.bounds,
+            collectionViewLayout: UICollectionViewCompositionalLayout.list(using: UICollectionLayoutListConfiguration(appearance: .plain)))
+        return collectionV
+    }()
+
+    private lazy var dataSource: UICollectionViewDiffableDataSource<SectionTitles, Brands> = {
+        let cellRegistration = UICollectionView.CellRegistration<UICollectionViewListCell, Brands> { cell, index, res  in
+            var contentConfig = cell.defaultContentConfiguration()
+            contentConfig.text = res.name
+            cell.contentConfiguration = contentConfig
+        }
+
+        return UICollectionViewDiffableDataSource(collectionView: brandsCollection) { collectionView, indexPath, itemIdentifier in
+            collectionView.dequeueConfiguredReusableCell(using: cellRegistration, for: indexPath, item: itemIdentifier)
+        }
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.addSubview(brandsCollection)
+        brandsCollection.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
should move up no need to split. whole config code for ui component would be in one modular place then